pkg/gds/client/profiles.go

```diff
@@ -50,37 +50,43 @@ const (
 func DefaultProfiles() *Profiles {
 	return &Profiles{
 		Version: ProfileVersion,
-		Active:  "testnet",
+		Active:  "localhost",
```

Comment:
Another few changes not strictly related to this PR but just to devEx; hope you don't mind me including them here.

Reply:
I think this is very helpful since the primary use case is development and the secondary use case is managing the network - thank you for including them here!